### PR TITLE
feat: use rc-qrcode instead of qrcode.react

### PR DESCRIPTION
**Pull request by kiner-tang**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | use rc-qrcode instead of qrcode.react |
| 🇨🇳 Chinese | 使用 rc-qrcode 替换 qrcode.react |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
解决一下冲突，再看一次CI，过了就搞正式的？

File: package.json [hunk 1]
```diff
@@ -147,6 +146,7 @@
     "rc-pagination": "~4.0.4",
     "rc-picker": "~4.5.0",
     "rc-progress": "~4.0.0",
+    "rc-qrcode-kiner": "1.0.2",
```
Comment (crazyair) [inline package.json:149]: 还带测试的啊

**Comment by afc163** [comment]
居然体积变大了

**Comment by kiner-tang** [comment]
> 居然体积变大了

原本的第三方库的代码不太符合 antd 的规范，所以这边稍微调整了一下结构，不过内部实现逻辑没有变化。

**Review by zombieJ — APPROVED** [review]

**Pull request merged**